alicloud/data_source_alicloud_kms_ciphertext_test.go

```diff
@@ -0,0 +1,46 @@
+package alicloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAlicloudKmsCiphertextDataSource(t *testing.T) {
+	resourceId := "data.alicloud_kms_ciphertext.default"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		// module name
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDataSourceKmsCiphertextConfigBasic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(
+						resourceId, "ciphertext_blob",
+					),
+				),
+			},
+			{
+				ResourceName:      resourceId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
```

Comment:
There is no need to test `import`, so the line 28-32 should be removed.

Reply:
Removed